src/basic/verbs.c

```diff
@@ -84,8 +112,11 @@ int dispatch_verb(int argc, char *argv[], const Verb verbs[], void *userdata) {
                 return -EINVAL;
         }
 
-        if ((verb->flags & VERB_NOCHROOT) && running_in_chroot() > 0) {
-                log_info("Running in chroot, ignoring request.");
+        if ((verb->flags & VERB_NOCHROOT) && running_in_chroot_or_offline() > 0) {
```

Comment:
If keeping `running_in_chroot_or_offline()` to bool, then `> 0` here should be remove.